src/applications/pensions/config/chapters/03-health-and-employment-information/helpers.js

```diff
@@ -81,9 +82,11 @@ export const generateMedicalCentersSchemas = (
 ) => {
   return {
     uiSchema: {
-      ...titleUI(medicalCentersTitle),
+      ...titleUI(
+        medicalCentersTitle,
+        <ArrayDescription message={medicalCenterMessage} />,
+      ),
```

Comment:
I don't think this 'ArrayDescription' component is necessary. We can also pass the description as a string into the 'titleUI' pattern and it will output the string as a span that is consistently styled with the other titles and descriptions throughout the form and have similar results to be announced by the screen reader.

```suggestion
      ...titleUI(
        medicalCentersTitle, medicalCenterMessage
      ),
```

If 'ArrayDescription' is specifically to override the existing description styles and not for the screen reader output, then it is fine.

Reply:
Yes, just for overriding the description styles to maintain the existing styling 👍